tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -1820,6 +1820,25 @@ def test_set_inbox(self):
         assert GroupInbox.objects.filter(group=group1).exists()
         assert not GroupInbox.objects.filter(group=group2).exists()
 
+    def test_set_resolved_inbox(self):
+        group1 = self.create_group(checksum="a" * 32)
+        group2 = self.create_group(checksum="b" * 32)
+
+        self.login_as(user=self.user)
+        with self.feature("organizations:inbox"):
+            response = self.get_valid_response(
+                qs_params={"id": [group1.id, group2.id]}, status="resolved"
+            )
+        assert response.data["inbox"] is False
```

Comment:
Need to change this to `None` too
```suggestion
        assert response.data["inbox"] is None
```

Reply:
I shouldn't have approved